sourcegraph/sourcegraph
# App: Update setup flow

Closes https://github.com/sourcegraph/sourcegraph/issues/52639
Part of https://github.com/sourcegraph/sourcegraph/issues/53419
[Designs](https://www.figma.com/file/ZkJFUvVzZW9TgFg7k9w9Oq/App-%5BMAIN%5D?type=design&node-id=76-887&t=mpxs6qahM81WI1BP-0)
 
## Todo 
- [x] Update visual styles for the setup steps pages 
- [x] Add up to 10 local repositories logic
- [x] Add external extension configuration load (it's done but there is a problem with cors that we have to solve before merging this PR)
- [x] Figure out right URLs for the extension repo and "suggest your extension" link
- [x] Hide Cody extension banner and survey toast UI in the app build

## Test plan
- Check visual styles based on new setup flow [designs](https://www.figma.com/file/ZkJFUvVzZW9TgFg7k9w9Oq/App-%5BMAIN%5D?type=design&node-id=76-887&t=mpxs6qahM81WI1BP-0) (note that progress bar will be implemented in a follow up PR)
- Check that you can add up to 10 local repositories on the embeddings step
- Check that extension page works properly (renders list of extension and provide right URLs for extension and its doc/repo page)
